src/Kestrel.Core/Internal/HttpConnection.cs

```diff
@@ -381,7 +377,7 @@ private HttpProtocols SelectProtocol()
             return http2Enabled && (!hasTls || Http2Id.Span.SequenceEqual(applicationProtocol.Span)) ? HttpProtocols.Http2 : HttpProtocols.Http1;
         }
 
-        public void Tick(DateTimeOffset now)
+        public void Tick(in DateTimeOffset now)
```

Comment:
Not sure this matters. I can change it back.

Reply:
Is a 16 byte struct, so will be copied and passed on stack without `in`; however also not a `readonly struct` 😢 so added PR for it  dotnet/coreclr#19552

Reply:
Err, will change it back.